src/domain/checkForAndPlaceOrder.ts

```diff
@@ -263,6 +263,8 @@ async function _processConditionalOrder(
         blockNumber,
       },
       provider,
+      // TODO: This should be DRY'ed. Upstream should take just an `orderBook` object that
```

Comment:
what do you mean `Upstream` here?

Reply:
`PollParams` should not take `orderbookApiConfig`, but should instead just take `orderBook` which is an instantiated Api with the parameters already set. Just like how the provider is already preset.

Reply:
Tracked in https://github.com/cowprotocol/cow-sdk/issues/181